packages/tasks/src/model-libraries.ts

```diff
@@ -317,6 +317,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://hezarai.github.io/hezar",
 		countDownloads: `path:"model_config.yaml" OR path:"embedding/embedding_config.yaml"`,
 	},
+	htrflow: {
```

Comment:
Hi @Borg93 - Thanks for the PR, I don't see any models with tags `htrflow`. Once this PR is in-effect it on model repos with the aforementioned tag (`htrflow`).

Reply:
Sure! Added some now : https://huggingface.co/models?other=htrflow